sys-devel/gettext/gettext-0.19.7.recipe

```diff
@@ -97,7 +107,6 @@ INSTALL()
 	make install AUTOCONF=: AUTOMAKE=: AUTOHEADER=: ACLOCAL=:
 
 	rm $libDir/charset.alias
-	rm $libDir/lib*.la
```

Comment:
IIRC, I even tried using **`--disable-static`** together with **`--with-pic=libintl`** in mc before dropping the **`rm $libDir/lib*.la`** in gettext, but I did not manage to make mc build without libintl.la. I guess I missed something.
Anyway, since **`lib*.la`** files are really small it is probably worth keeping a given **`lib*.la`** if we know that at least one recipe needs that particular file.
BTW, is there a tool to dump the list of shared libs that a runtime needs? On Linux I use ldd, but I did not find it on Haiku. Thanks!